### PR TITLE
put setup instructions into README

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,57 @@
...
+$apiSecret = 'mySecret';
+
+$imageClient = Factory::getImageClient($organization, $apiKey, $apiSecret);
+```
+
+There is an optional fourth parameter to specify the base URL of the Rokka API. This usually does not need to be adjusted.
+
+## Usage
+
+See the [official documentation](https://rokka.io/documentation) on how to use the Rokka API.
```
Comment (michellesanver) [inline README.md:53]: This is a bit confusing for the user, but that means we should probably change the rokka.io documentation, if we do that this line can stay.

File: README.md [hunk 2]
```diff
@@ -1,11 +1,57 @@
...
-Run `vendor/bin/phpunit` from project root
+Require the library using composer:
+
+`composer require rokka/client`
+
+## Bootstrapping
+
+The `Rokka\Client\Factory` is the entry point for creating the API client.
+
+You will need to create an account on Rokka to get your api key and secret and create an organization.
```
Comment (michellesanver) [inline README.md:17]: In this sprint the organization will be created automatically when you sign up. So maybe the best line here would be "You will need to register for a Rokka account and use the api key and secret that you recieved" it doesn't have to mention the "create an organization" step.

**Review by michellesanver — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (michellesanver) [reply README.md:53]: Actually could change this line to "on how to use the Rokka API with curl examples" for now and open up a ticket to make PHP examples in that doc and then change this line.

**Comment by chregu** [comment]
lots of php code examples are also on rokka.io/documentatino, btw...

File: README.md (hunk 1 above)
Comment (dbu) [reply README.md:53]: there are php examples e.g. on https://rokka.io/documentation/references/memberships.html - imo that is good enough for now. i would merge the readme as is and then work on the documentation side at some point. for example the tabs approach for code examples.

**Comment by dbu** [comment]
no idea what was going on with github here, had to re-create the PR as #21

**Pull request closed without merging**